### DISCUSSION TITLE
How to reset GameHub configuration?

### DISCUSSION THREAD

**Discussion by epassaro** [tkashkin/GameHub#589] [Q&A] [answered]
Hi,

I'll tried to reset the GameHub configuration by deleting files in `.config/` `.local` and `.cache`, but GameHub is still logged to GOG and Steam.

I want to delete all the configuration files, I'm not interested in doing this from the UI.

**Comment by tkashkin** [comment, marked as answer]
Run `dconf reset -f /com/github/tkashkin/gamehub/` or reset GameHub settings in `dconf-editor`.

**Reply by epassaro** [reply]
Thanks!
